migrations/2_deploy_token.js

```diff
@@ -10,8 +10,13 @@ const log = require('./utils/log');
 const NativeToken = artifacts.require('NativeToken');
 
 module.exports = (deployer) => {
+
+  const deployedToken = process.env.DEPLOYED_TOKEN;
+
   deployer.then(async () => {
-    const nativeToken = await deployer.deploy(NativeToken, "LeapToken", "LEAP", 18);
-    log('Deployed LEAP Token at', nativeToken.address);
+  	if (!deployedToken) {
```

Comment:
still not sure this is the right way to use truffle migrations

Reply:
agree, but reading through migration contract i didn't get any wiser 🤷‍♂️